core-bundle/src/Resources/contao/themes/flexible/main.css

```diff
@@ -35,10 +35,14 @@ body:not(.fullscreen) #header .inner {
 }
 #header h1 a {
 	display:block;
+	height:16px;
 	padding:12px 12px 12px 43px;
 	background:url("icons/logo.svg") no-repeat 10px center;
 	font-weight:400;
 }
+#header h1 a .app-title {
+	float: left;
```

Comment:
Is this really necessary? IMHO only the badge needs to float.

Reply:
It is neccesary for IE 11 support - see without float in IE: 
![image](https://user-images.githubusercontent.com/14016098/104003885-55723800-51a3-11eb-91ea-19ac2a30049e.png)

Reply:
You could change the order in the template but I wanted the app-title "Contao" as first before the badge-title so you need to float the app-title